dashboard/lib/standards_autocomplete.rb

```diff
@@ -0,0 +1,18 @@
+class StandardsAutocomplete < AutocompleteHelper
+  def self.get_search_matches(query, limit, framework_id = nil)
+    limit = format_limit(limit)
+    rows = Standard.limit(limit)
+    rows = rows.where(framework_id: framework_id) if framework_id
+
+    shortcode_rows = rows.where('shortcode like ?', "#{query}%").all
+    return shortcode_rows.map(&:summarize_for_lesson_edit) unless shortcode_rows.empty?
```

Comment:
I'm having trouble finding it in the mysql docs, but this page suggests that a wildcard at the end of a LIKE query can still make full use of the index: https://use-the-index-luke.com/sql/where-clause/searching-for-ranges/like-performance-tuning